hack/build/gosec.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/bash
```

Comment:
There are still references to this file in the Makefile, is this intentional?

Reply:
No :sweat_smile: I got rid of it now